src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -154,8 +154,8 @@ which we divide into several categories:
 
 {{ MACROS___make_metadata_table(
    {
-      "TracerName": "REQUIRED",
-      "TracerRadionuclide": "REQUIRED",
+      "TracerName": ("REQUIRED", "Corresponds to DICOM Tags (0008,0105) `Mapping Resource` and (0008,0122) `Mapping Resource Name`."),
```

Comment:
Why aren't we putting these in the metadata.yaml directly?

Reply:
well some were in yaml some were in spec, and there is no rule to decide --so being lazy i pushed all new ones there

Reply:
The rule I would use is whether the BIDS term unambiguously corresponds to a DICOM tag (so would be sensible to render any time the term is rendered), or if it corresponds to a DICOM tag in the specific context of a particular table.

Reply:
I'm not confident enough to say that these tags should belong in the `metadata.yml` as many of these terms are often more explicitly defined (or known....) by the experiment designer/performer or simply not present in the dicom headers. Adding them to the tables provides a bread crumb for a user to extract this info if they're not omnipotent concerning the experiment or the metadata.

Reply:
"Corresponds to DICOM Tag" doesn't say to me "MUST be derived from DICOM Tag". I think linking the concepts in the term definition helps clarify.

That said, I won't die on this hill. We can move these into `metadata.yaml` in the future as well.

Reply:
> "Corresponds to DICOM Tag" doesn't say to me "MUST be derived from DICOM Tag". I think linking the concepts in the term definition helps clarify.
> 
> That said, I won't die on this hill. We can move these into `metadata.yaml` in the future as well.

Agreed. Have already noticed in this PR that it's very easy 🙄 to add something into these tables and duplicate what's in the `metadata.yaml` or vice versa.